hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -294,3 +296,103 @@ export const PendingTable = ({ items, status }) => {
         </Container>
     );
 };
+
+export const UnconnectedBrokenTable = ({
+    items,
+    isVisible,
+    toggleVisibility,
+    onClickViewReport,
+    status,
```

Comment:
You don't need to pass in a `status` prop. The Orders Pending table has a status for the state of the order (order is ready, order is still is progress, error with order).